### PR TITLE
Record error

### PR DESCRIPTION
**Pull request by jburel**
Record error
Check that the application still runs 

cc @mtbc

### CODE REVIEW

File: src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java [hunk 1]
```diff
@@ -473,14 +473,9 @@ public void connect(UserCredentials uc)
 	    
 		if (uc == null)
             throw new NullPointerException("No user credentials.");
+        LogMessage msg;
 		String name = (String) 
 		container.getRegistry().lookup(LookupNames.MASTER);
-		if (CommonsLangUtils.isBlank(name)) {
-            name = LookupNames.MASTER_INSIGHT;
```
Comment (mtbc) [inline src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: It's okay to move this step even though `cred.setApplicationName` won't then benefit from it?

Comment (jburel) [reply src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: Looking at the previous code, it was set as OMERO.insight and not insight that's why I moved it

Comment (jburel) [reply src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: sorry that line should be kept where it was

**Comment by mtbc** [comment]
Is https://github.com/ome/omero-insight/pull/46/files#diff-d3089c1dff7a3f6c789980fcb8bcb74aL581 broken? I'm starting to wonder if safest would be to put the no-"OMERO." version into a different variable for the upgrade check rather than mutating `name` itself.

File: src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java [hunk 2]
```diff
@@ -490,8 +488,13 @@ public void connect(UserCredentials uc)
         cred.setCheckNetwork(true);
         cred.setCompression(determineCompression(uc.getSpeedLevel()));
         cred.setEncryption(uc.isEncrypted());
-        
-		ExperimenterData exp = omeroGateway.connect(cred);
+        ExperimenterData exp = null;
+        try {
+            exp = omeroGateway.connect(cred);
+        } catch(Exception e) {
+            e.printStackTrace();
```
Comment (mtbc) [inline src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: Is this intended?

Comment (jburel) [reply src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: no, I was trying to figure out why I could not connect to merge-ci

**Pull request merged**